node/app.js

```diff
@@ -30,48 +30,52 @@ app.set('view engine', 'njk');
 
 app.get('/', function(req, res){
    session_variables = req.session;
-   if(session_variables.email){
-      console.log("Email: " + session_variables.email);
-      res.render('index', { email : session_variables.email});
+   if(session_variables.Employee_ID){
```

Comment:
Consider using a [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to simplify code here.